tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/update_invalid_zip_codes_test.py

```diff
@@ -112,31 +107,35 @@ def test_update_invalid_zip_codes(self):
                 -- valid zip codes, will not be affected --
                 (1, 1, 0, date('2019-03-03'), 0, 1585250, null, '12345', null, null), 
                 (2, 2, 0, date('2019-03-03'), 0, 1585250, null, '23456-1234', null, null),
-                
+
                 -- invalid zip codes with less than 5 digits, will be sandboxed and updated in observation table --
                 (3, 3, 0, date('2019-03-03'), 0, 1585250, null, '1234', null, null), 
                 (4, 4, 0, date('2019-03-03'), 0, 1585250, null, '123', null, null),
                 (5, 5, 0, date('2019-03-03'), 0, 1585250, null, '12', null, null),
                 (6, 6, 0, date('2019-03-03'), 0, 1585250, null, '1', null, null),
-                
+
                 -- not zip codes (not zip code concept id), will not be affected --
                 (7, 7, 0, date('2019-03-03'), 0, 1111111, null, '111', null, null),
                 (8, 8, 0, date('2019-03-03'), 0, 2222222, null, '222', null, null),
-                
+
                 -- invalid zip codes with 5 characters and unknown zip 3, will be sandboxed and updated in obs table --
                 (9, 9, 0, date('2019-03-03'), 0, 1585250, null, '00501', null, null),
                 (10, 10, 0, date('2019-03-03'), 0, 1585250, null, '00566', null, null),
-                
+
                 -- invalid zip codes, are alphanumeric, will be sandboxed and updated in observation table --
                 (11, 11, 0, date('2019-03-03'), 0, 1585250, null, '123ab', null, null),
                 (12, 12, 0, date('2019-03-03'), 0, 1585250, null, '1abc', null, null),
                 (13, 13, 0, date('2019-03-03'), 0, 1585250, null, 'abc12', null, null), 
-                
+
                 -- invalid zip codes, excess whitespace, will be sandboxed and updated in observation table --
                 -- once cleaned will be valid --
                 (14, 14, 0, date('2019-03-03'), 0, 1585250, null, '     34567', null, null), 
                 -- once cleaned will be invalid --
-                (15, 15, 0, date('2019-03-03'), 0, 1585250, null, '56789     ', null, null) 
+                (15, 15, 0, date('2019-03-03'), 0, 1585250, null, '56789     ', null, null), 
+
+                -- valid zip codes with preceding 0s will not be affected --
+                (16, 16, 0, date('2019-03-03'), 0, 1585250, null, '02035', null, null),
+                (17, 17, 0, date('2019-03-03'), 0, 1585250, null, '00601', null, null)
```

Comment:
I understand that `12345` is a valid zip code, and `12345-6789` is also valid. Is `123456789`(zip code without hyphen) valid or invalid? I saw 9-digit codes in the discussion in the comments from `DC-2739`(the investigation ticket). I think it is worth adding that test case to this test to clarify the expected behavior.